src/consts.js

```diff
@@ -47,11 +49,32 @@ const TASK_OUTPUT_FIELDS = [
     { key: 'defaultRequestQueueId', label: 'Default request queue ID', type: 'string' },
 ];
 
+const TASK_RUN_SAMPLE = Object.assign({ actorTaskId: 'UJNG9zau8PEB7U' }, ACTOR_RUN_SAMPLE);
+
+const TASK_RUN_OUTPUT_FIELDS = ACTOR_RUN_OUTPUT_FIELDS.concat([{ key: 'actorTaskId', label: 'Actor task ID', type: 'string' }]);
+
 const DEFAULT_KEY_VALUE_STORE_KEYS = ['OUTPUT'];
 
+const DEFAULT_PAGINATION_LIMIT = 100;
+
+// Actor ID of apify/legacy-phantomjs-crawler
+const LEGACY_PHANTOM_JS_CRAWLER_ID = 'YPh5JENjSSR6vBf2E';
```

Comment:
We should move this to shared also. Ill try to keep it in mind when I do some consts changes.